src/Tools/dotnet-monitor/Auth/LiveJwtBearerHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+
+namespace Microsoft.Diagnostics.Tools.Monitor
+{
+    /// <summary>
+    /// This is a version of <see cref="JwtBearerHandler" /> that will refresh the 
+    /// stored JwtBearerOptionsoptions when the settings change.
```

Comment:
I'm assuming this should be either "JwtBearerOptions" or "JwtBearerOptions options"